apps/web/src/app/challenge/_components/challenge-layout.tsx

```diff
@@ -80,9 +80,25 @@ export function ChallengeLayout({ left, right }: ChallengeLayoutProps) {
       const newLeftWidth = ((leftWidth + dx) * 100) / divideByW;
       const newTopHeight = ((topHeight + dy) * 100) / divideByH;
 
-      window.innerWidth > 1025
-        ? (leftRef.style.width = `${newLeftWidth}%`)
-        : (leftRef.style.height = `${newTopHeight}%`);
+      if (window.innerWidth > 1025) {
```

Comment:
lets avoid magic numbers and assign these to const vars

Reply:
> lets avoid magic numbers and assign these to const vars

it's the `md:` screen size from https://github.com/bautistaaa/typehero/blob/9ec2e9c813c940df5f2387a1d355713ac16f8372/apps/web/tailwind.config.ts#L17C3-L17C22

Reply:
thats fine. just need to assign it to a meaningful name

Reply:
@PickleNik does TW give you a way to import `lg` as a value from the theme in js land?